src/UNIV2LPOracle.t.sol

```diff
@@ -416,6 +444,15 @@ contract UNIV2LPOracleTest is DSTest {
         seekableOracleDAI._seek();                                                // Get new dai-eth lp price from uniswap
     }
 
+    function testFail_seek_zero_LPToken_supply_wbtc() public {
+        hevm.store(
+            WBTC_ETH_UNI_POOL,
+            0,                                                                    // totalSupply
+            bytes32(uint256(0))
+        );
+        seekableOracleWBTC._seek();                                                // Get new wbtc-eth lp price from uniswap
+    }
+
     function test_seek_internals() public {
```

Comment:
I think this entire test can be removed now that the others have better coverage for the expected result. It hasn't even been kept in-sync with the internal logic it was originally testing.

Reply:
It's the only place where some important sanity checks are being performed, such as the accuracy of the token reserves, their decimal representations, the behavior of `sync`, and the calculation of the quote in terms of K.

As production code relies on some results of these tests, I think we should update them instead instead of removing them. What needs to be updated?

Reply:
But it's not testing the production code at all (except for the one line you added). If any of those steps were wrong, the other tests in this file would fail. This test is an anti-pattern.